### ISSUE TITLE
Issue with Demo.R for Jan 2017 TCRUG talk

### ISSUE THREAD

**Issue by Nicholaev** [cpsievert/talks#1]
Carson. Thanks to your, I finally found the missing piece I've been looking for in R, the interactive piece. I was trying out your demo code but found an error when running lines 31 and 32:
`p <- ggplot(tx) +
    geom_line(aes(x = date, y = median, group = city))`
I got an error: 
  `ggplot2 doesn't know how to deal with data of class SharedData/R6. `
Installation of both the plotly and crosstalk libraries were error free. Codes above these lines ran fine. Codes following them have similar error wrt the ShareData class. I haven't used Crosstalk before. What would you suggest? I've restarted the computer and reloaded both libraries. The error did not change. 
Regards,

**Comment by cpsievert** [comment]
Make sure you have the *dev* version of plotly loaded before running that line

**Comment by Nicholaev** [comment]
Oh, didn't know that. I googled this and found a github post that suggested using 
`devtools::install_github("ropensci/plotly")`
to install the development version. I did that and got an error 
`Error in loadNamespace(name) : there is no package called ‘devtools’`
Have you seen this problem? The post said to make sure the latest Rstudio is used. Mine is the latest version.

**Comment by Nicholaev** [comment]
Alright! I found the complete instruction in another Github posting. I need to install devtools first. Did that. 
Now got an error during plotly installtion:

`ERROR:dependency 'digest' is not available for package 'plotly'`
because cannot delete an earlier version of digest. Went in the library folder and deleted the digest folder. Ran the installation again. This time got a different error:

`Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called 'miniUI'
ERROR: lazy loading failed for package 'plotly'`

Installed miniUI. It worked! Only partly though. Lines 31 and 32 now recognizes the SharedData class. Still cannot plot consistently using plotly. Some of the codes worked fine. Others such as this

`p <- ggplot(txhousing) +
  geom_line(aes(x = date, y = median, group = city))
p %>% ggplotly()`
had error: 

`Error in ggfun("create_layout")(plot$facet, plot$coordinates) : 
  unused argument (plot$coordinates)`

I wonder if this is the same issue you had during the talk?
